packages/cli/src/webhooks/webhook-form-data.ts

```diff
@@ -0,0 +1,44 @@
+import formidable from 'formidable';
+import type { IncomingMessage } from 'node:http';
+
+type IncomingMessageWithEncoding = IncomingMessage & { encoding: string };
```

Comment:
`IncomingMessage` is already augmented [here](https://github.com/n8n-io/n8n/blob/master/packages/workflow/src/index.ts#L66).